composer.json

```diff
@@ -31,7 +31,7 @@
         "phpspec/prophecy": "^1.6.2",
         "phpunit/phpunit": "^5.7.25|^6.4.4",
         "sebastian/comparator": "^1.2.4",
-        "sensiolabs/security-checker": "^3.0",
+        "sensiolabs/security-checker": "^3.0|^4.0",
```

Comment:
why not only v4 ?

Reply:
Good point. 😉 Didn't really look into its dependency chain.
And also turns out this is only for `dev-dependencies`, too. So yeah...